### ISSUE TITLE
Bug? Data API v3:  GET entries endpoint has inconsistent response

### ISSUE THREAD

**Issue by nakedsushi** [movabletype/movabletype#369]
version `r.4208 (7.0.1)`

Doing the same GET request to `/mt/mt-data-api.cgi/v3/sites/1/entries?fields=author,basename` will return different responses even if no entries were added or deleted.

Steps to reproduce:
1. Make an GET request to ` /mt/mt-data-api.cgi/v3/sites/1/entries?fields=author,basename`, take note of the response.
2. Make the same request again. Look at the response, the same number of items are returned, but the parameters in the items are usually missing or different.

I don't think it's a memcached error because I have disabled memcached and this still keeps happening.

**Comment by miniuchi** [comment]
@nakedsushi 
Thank you for your comment. I could not reproduce it.
Does your MT use 3rd party's plugin or addon? If so, please disable them and check again.

**Comment by nakedsushi** [comment]
More clues. I wonder if it's the way starman is set up? When the bug starts happening, if I restart starman with `systemctl restart movabletype.service` then it goes away -- until the next time I insert a bunch of entries via the API.

**Comment by miniuchi** [comment]
@nakedsushi 
Thank you. I will investigate it again on the AMI.

**Comment by miniuchi** [comment]
@nakedsushi 
I couldn't reproduce it on AMI... Could you give me more detail procedure to reproduce it if possible?

**Comment by nakedsushi** [comment]
Steps to reproduce:

1. Set up `/data/conf/movabletype.conf` like this:
```
WORKER=2
MAXREQUESTS=10
```
1. Restart the nginx service with `systemctl restart nginx.service`
1. Create a script to post 1000 new entries to the DataAPI entry endpoint here: `https://www.movabletype.jp/developers/data-api/v3-reference.html#entries`

1. Run the script so that it does the POST requests serially.

1. BUG: some of the entries are correctly created, others have blank titles and bodies.

1. BUG: after the bug above happens, try listing the entries with the list entries endpoint:
`../mt/mt-data-api.cgi/v3/sites/SITE_ID/entries` sometimes all the fields for the entries are returned and sometimes they are blank.

**Comment by miniuchi** [comment]
@nakedsushi 
Thank you.

I tried to reproduce it with the following script and MT7 nginx AMI (t2.medium), but couldn't.
https://gist.github.com/miniuchi/3551252947985fd6dc4c5baecc306df5

I wrote this script with reference to https://github.com/movabletype/movabletype/issues/369#issuecomment-398091157.
Am I misunderstanding your procedure?

**Comment by nakedsushi** [comment]
@miniuchi You're right, I can't reproduce it now either on my AMI with your script. I'll try it again when I get back into work with my full script and see what the difference is. Thanks for working on this!

**Comment by nakedsushi** [comment]
Update: I haven't been able to consistently reproduce it with my own script. Sometimes it happens, sometimes it doesn't and setting the number of MAXREQUEST to something low like 10 sometimes works and doesn't.  I'll play around with it some more and see if I can get more consistency with it.

**Comment by miniuchi** [comment]
@nakedsushi 
Thank you for your investigation!
